ledger/complete/mtrie/flattener/iterator_test.go

```diff
@@ -7,36 +7,33 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/onflow/flow-go/ledger"
-	"github.com/onflow/flow-go/ledger/common/utils"
 	"github.com/onflow/flow-go/ledger/complete/mtrie/flattener"
 	"github.com/onflow/flow-go/ledger/complete/mtrie/trie"
 )
 
 func TestEmptyTrie(t *testing.T) {
-	emptyTrie, err := trie.NewEmptyMTrie(1)
-	require.NoError(t, err)
+	emptyTrie := trie.NewEmptyMTrie()
 
 	itr := flattener.NewNodeIterator(emptyTrie)
 	require.True(t, nil == itr.Value()) // initial iterator should return nil
 
-	require.True(t, itr.Next())
+	require.False(t, itr.Next())
```

Comment:
Is this expected ?

Reply:
I believe so. Tarak stated in his PR description:
> an empty trie is now defined as a trie with a nil root node, instead of a trie with a non nil root node with a nil path.

Hence, in the previous version, the empty Trie would still have a node with default hash. In the new implementation, there is no root node anymore. Hence, already the very first call of `itr.Next()` returns `false`.

Reply:
Yep, I confirm that's the expectation of the updated test.